build.gradle

```diff
@@ -383,13 +383,11 @@ project('spring-rabbit-test') {
 	dependencies {
 
 		compile project(':spring-rabbit')
+		compile project(':spring-rabbit-junit')
 		compile "org.hamcrest:hamcrest-library:$hamcrestVersion"
 		compile "org.hamcrest:hamcrest-core:$hamcrestVersion"
-		compile ("org.mockito:mockito-core:$mockitoVersion") {
-			exclude group: 'org.hamcrest', module: 'hamcrest-core'
-		}
+		compile "org.mockito:mockito-core:$mockitoVersion"
 		testCompile project(':spring-rabbit').sourceSets.test.output
-		testCompile project(':spring-rabbit-junit')
 	}
 
```

Comment:
We should add something to the What's New - people who are using TestNG will now have to exclude it.

Reply:
Well, not sure how does that affect.
A `junit` and `junit-jupiter-api` are *optional* in the `spring-rabbit-junit`, but we indeed can add a note in What's New.

Reply:
Oh, of course; never mind.